chain/network-primitives/Cargo.toml

```diff
@@ -1,27 +1,31 @@
 [package]
 name = "near-network-primitives"
 version = "0.0.0"
-authors = ["Near Inc <hello@nearprotocol.com>"]
-publish = true
+authors = [
+    "Near Inc <hello@nearprotocol.com>",
+    "Piotr Mikulski <piotr@near.org>"
```

Comment:
Not sure about NEAR - but normally you'd not mention your name directly as the owner of the packet (this becomes stale too often)